packages/apps/job-launcher/client/src/providers/CreateJobPageUIProvider.tsx

```diff
@@ -55,11 +57,15 @@ export const CreateJobPageUIProvider = ({
   });
 
   const goToPrevStep = () => {
-    setStep((prev) => prev - 1);
+    if (step > CreateJobStep.FundingMethod) {
+      setStep((prev) => prev - 1);
+    }
   };
 
   const goToNextStep = () => {
-    setStep((prev) => prev + 1);
+    if (step < CreateJobStep.PayJob) {
```

Comment:
Not sure, but just to clarify, shouldn't it be `< CreateJobStep.Launch`? Or is the last step automatic?

Reply:
oh, good catch